transpiler/src/zero_generator.rs

```diff
@@ -198,6 +210,22 @@ impl Transpiler {
         self.add_instruction(move_inst);
     }
 
+    fn transpile_if_statement(&mut self, op: &ExprIfStatement) {
+        self.transpile_op(&op.first_expr);
+        let prop = self.current_stack_frame.evaluation_stack.offset_of_last_element();
+        let dest0 = self.new_label();
+        let dest1 = dest0.clone();
```

Comment:
I think it's more idiomatic to just make one `dest`, and then call `.clone()` on it when necessary as its used